lib/OpenQA/Worker/Cache.pm

```diff
@@ -306,7 +306,7 @@ sub update_asset {
         $dbh->rollback;
     }
     else {
-        log_info "CACHE: updating the $asset with $etag and $size";
+        log_debug "CACHE: updating the $asset with $etag and $size";
```

Comment:
This message could be at info level, since size could be different (as it is calculated once the asset has finished downloading)